dev/devicelab/lib/framework/framework.dart

```diff
@@ -143,7 +143,7 @@ class _TaskRunner {
     });
   }
 
-  /// Disables the keep-alive port, allowing the VM to exit.
+  /// Disables the keepalive port, allowing the VM to exit.
```

Comment:
```suggestion
  /// Disables the [keepAlivePort] allowing the VM to exit.
```

Reply:
That would be great, but it's a private member: `_keepAlivePort`, and there isn't any description to link to.